<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>check</class>
 <widget class="QDialog" name="check">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Validate ISBN</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/validation.png</normaloff>:/validation.png</iconset>
  </property>
  <widget class="QLineEdit" name="lineEditNum">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>275</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxType">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>275</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>ISBN - 10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ISBN - 13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Local Visa/MasterCard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>International Visa/MasterCard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Universal Product Code</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="number_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Enter Number: </string>
   </property>
  </widget>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose Type:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: black;
    color: white;
    border-radius: 10px;
    text-align: center;
}</string>
   </property>
   <property name="text">
    <string>Validate</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
